backend/config/config.go

```diff
@@ -315,6 +324,10 @@ func ParseNomadFlagConfig(c *Config) {
 	if *flagNomadAllowStale {
 		c.NomadAllowStale = *flagNomadAllowStale
 	}
+
+        if *flagNomadDisableFS {
```

Comment:
Is `flagNomadDisableFS` a ptr? Won't dereferencing this cause a crash if `flagNomadDisableFS` ends up being nil?